GoogleMapsApi.Test/IntegrationTests/GeocodingTests.cs

```diff
@@ -33,94 +32,53 @@ public void Geocoding_ReturnsCorrectLocation()
 
 
         [Test]
-        public void GeocodingAsync_ReturnsCorrectLocation()
```

Comment:
Again, have a look at the sync version of that test, it will be the same as the test that was removed.